pkg/tfbridge/README.md

```diff
@@ -0,0 +1,148 @@
+# tfbridge
+
+## Automatic Aliasing (`ApplyAutoAliases`)
+
+Automatically applies backwards compatibility best practices.
+
+The goal is to prevent breaking changes from Pulumi maintainers or from the upstream
```

Comment:
```suggestion
The goal is to prevent breaking changes by Pulumi maintainers or by the upstream
```
Wanted to disambiguate the prepositions a bit here